ompi/mca/pml/base/pml_base_frame.c

```diff
@@ -100,12 +100,27 @@ char *ompi_pml_base_bsend_allocator_name = NULL;
 static char *ompi_pml_base_wrapper = NULL;
 #endif
 
+#if MPI_VERSION >= 4
+int ompi_pml_base_deprecation_warn_level = 1;
+mca_base_var_enum_value_t ompi_pml_base_deprecation_warnings[] = {
+    {1, "once"},
+    {2, "always"},
```

Comment:
Should we order this by verbosity: always, once, never?

Reply:
Yeah that's a good idea.